ports/imagestriker/imagestriker/gmloader.json

```diff
@@ -0,0 +1,7 @@
+{
+    "save_dir" : "save",
```

Comment:
`$GAMEDIR/save` doesn't exist, is that fine, does the game run ?

Reply:
The game creates it on its own along with the save.ini

Reply:
Oh great, I wasn't sure it would be created at run.